tutorials/xr/xr_action_map.rst

```diff
@@ -385,3 +385,124 @@ This is why many XR runtimes only use it as a last resort and will attempt to us
   This is our advice as well: limit your action map to the interaction profiles for devices you have actually tested your game with.
   The Oculus Touch controller is widely used as a fallback controller by many runtimes.
   If you are able to test your game using a Meta Rift or Quest and add this profile there is a high probability your game will work with other headsets.
+
+Binding Modifiers
+-----------------
+
+While one of the main goals of the action map is to remove the need for the application to know the hardware used, sometimes the hardware has physical differences that require inputs to be altered in ways other than how they are bound to actions. This need ranges from setting thresholds, to altering the inputs available on a controller.
+
+Currently OpenXR supports two binding modifiers that have been implemented in Godot but more are likely to follow in the future.
+
+These binding modifiers are not enabled by default and require enabling in the OpenXR project settings. Also there is no guarantee that these modifiers are supported by every runtime. You will need to consult the support for the runtimes you are targeting and decide whether to rely on the modifiers or implement some form of fallback mechanism.
+
+If you are targeting multiple runtimes that have support for the same controllers, you may need to create separate action maps for each runtime. You can control which action map Godot uses by using different export templates for each runtime and using a custom :ref:`feature tag <doc_feature_tags>` to set the action map.
+
+In Godot, Binding modifiers are divided into two groups. Modifiers that work on the interaction profile level, and modifiers that work on individual bindings.
+
+Binding modifiers on an interaction profile
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Binding modifiers that are applied to the whole interaction profile can be accessed through the modifier button on the right side of the interaction profile editor.
+
+.. image:: img/openxr_ip_binding_modifier.webp
+
+You can add a new modifier by pressing the ``Add binding modifier`` button.
+
+.. warning::
+  As Godot doesn't know which controllers and runtimes support a modifier,
+  there is no restriction to adding modifiers.
+  Unsupported modifiers will be filtered out at runtime.
+
+Dpad Binding modifier
```

Comment:
I don't have time to look thoroughly right now but "dpad" is a word we should check the standardization of. From a quick search I see "DPad", "dpad", and "D-pad" even with XR docs specifically

This can probably be solved in a future PR just making a note

Reply:
I'll change it to "dpad" for the OpenXR docs. Thats the name they are using in the binding modifier spec: https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#XR_EXT_dpad_binding

Reply:
Makes sense, seems fine to me.

At some point I might take a look at what we use in general when referring to the dpad and standardize some more. It may be the case that we end up using "DPad" or "D-pad" generally and only using "dpad" when referring to the spec.

Reply:
Yeah sounds good, it makes sense but it sounds like a lot of work :)